lock-api-objects/src/main/java/com/palantir/lock/watch/LockEvent.java

```diff
@@ -64,4 +135,56 @@ public static LockWatchEvent.Builder builder(
                 .metadata(metadata);
         return seq -> builder.sequence(seq).build();
     }
+
+    @Unsafe
+    @Value.Immutable
+    @JsonSerialize(as = ImmutableWireLockEvent.class)
+    @JsonDeserialize(as = ImmutableWireLockEvent.class)
```

Comment:
These are only used for serialisation, not deserialisation - let's make that explicit by not supporting deserialisation with these (and leaving a comment!)

Reply:
Also added a test to verify that the class cannot be used for deserialization